src/index.css

```diff
@@ -1,41 +1,251 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;400;700&display=swap');
+
 * {
   margin: 0;
   padding: 0;
   font-size: 100%;
+  font-size: 16px;
   vertical-align: baseline;
-  font-family: 'Open Sans', 'Helvetica Neue', sans-serif;
+  list-style: none;
+
+  --white: #fff;
+  --white-two: #fafafa;
+  --black-two: #121212;
+  --azure: #0290ff;
+  --border-color: #e8e8e8;
 }
 
-.book-form {
+.pointer:hover {
+  cursor: pointer;
+}
+
+.body {
+  background-color: var(--white-two);
+  min-height: 100vh;
+}
+
+.d-flex {
   display: flex;
```

Comment:
Your CSS appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code.

Let's improve our styling by using [Combine Elements](https://speckyboy.com/good-bad-css-practices)

Example:

You have `display: flex`, `  justify-content: space-between;
`, `  align-items: center;
` written more than twice, which is simply a repetition of rules;

You could have it this way;
```
.d-flex, .book-panal, .link-list, .progress-container {
    display: flex
}
```

OR
 
 Create a rule.
```
.someCommonStyle {
  display: flex
}
```
Then we apply this style to all the elements with this rule. Doing this will make our code cleaner and easier to maintain